plugins/mux/serve_interceptor.go

```diff
@@ -63,7 +63,7 @@ func newWriterWrapper(rw interface{}) *writerWrapper {
 	writer := rw.(http.ResponseWriter)
 	hijacker, ok := rw.(http.Hijacker)
 	if !ok {
-		log.Errorf("http.ResponseWriter does not implement http.Hijacker")
+		log.Warnf("http.ResponseWriter does not implement http.Hijacker")
```

Comment:
It's unnecessary because the Hijicker is not used for some frameworks. Users will be confused about this log.